src/netaddress.h

```diff
@@ -334,31 +330,22 @@ class CNetAddr
                 memcpy(arr, IPV4_IN_IPV6_PREFIX.data(), prefix_size);
                 memcpy(arr + prefix_size, m_addr.data(), m_addr.size());
                 return;
-            case NET_ONION:
```

Comment:
L301 mentions that `GetBIP155Network()` *Must not be called for IsInternal() objects.* 

Would it make sense to add a note about Tor v2 not being supported anymore? The reason why it may make sense is simply because [BIP155](https://github.com/bitcoin/bips/blob/master/bip-0155.mediawiki) is written with Tor v2 in mind (and that won't change, or will it?) and the function name is `GetBIP155Network`. So we are about to deviate from BIP155. For a good reason but still.

Reply:
The idea was to change:

```cpp
/**
 * Get the BIP155 network id of this address.
 * Must not be called for IsInternal() objects.
 * @returns BIP155 network id
 */
BIP155Network GetBIP155Network() const;
```

to something like

```cpp
/**
 * Get the BIP155 network id of this address.
 * Must not be called for IsInternal() objects.
 * @returns BIP155 network id except TORV2 which is no longer supported
 */
BIP155Network GetBIP155Network() const;
```

but it's probably an overkill.

Reply:
SGTM, will add to the follow-up refactoring commit unless this patch needs a more substantial update.

Reply:
Updated
```diff
-         * @returns BIP155 network id
+         * @returns BIP155 network id, except TORV2 which is no longer supported.
```